server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1819,8 +1820,19 @@ public JobDetail exportDataAsync(
         @PathParam("consumer_uuid") @Verify(Consumer.class) String consumerUuid,
         @QueryParam("cdn_label") String cdnLabel,
         @QueryParam("webapp_prefix") String webAppPrefix,
-        @QueryParam("api_url") String apiUrl) {
-        return manifestManager.generateManifestAsync(consumerUuid, cdnLabel, webAppPrefix, apiUrl);
+        @QueryParam("api_url") String apiUrl,
+        @QueryParam("ext") @CandlepinParam(type = KeyValueParameter.class)
```

Comment:
Needs Swagger attributes so that it is documented properly as to what the expected values are and their purpose.